src/variety/country.js

```diff
@@ -165,15 +176,20 @@
 			}
 		}
 	},
+	"Cell@dotchi#1": {
```

Comment:
It’s been a while since I was familiar with the object system of `pzprjs`. Can you remind me what `#1` signifies?

Reply:
If I leave out the `#1`, this declaration would conflict with `Cell@moonsun,dotchi` above. pzprjs strips the puzzle names after the `@`.

Reply:
Ah, I understand now. Thanks for the explanation.